src/cmd-buildextend-installer

```diff
@@ -71,6 +74,10 @@ os.mkdir(tmpisoroot)
 os.mkdir(tmpisoimages)
 os.mkdir(tmpisoisolinux)
 
+# Number of padding bytes at the end of the ISO initramfs for embedding
+# an Ignition config
+initrd_ignition_padding = 256 << 10
```

Comment:
This would be more human-friendly as `256 * 1024`.

Reply:
I find this phrasing _more_ readable, but sure.